packages/gatsby-transformer-screenshot/lambda/index.js

```diff
@@ -56,7 +56,7 @@ exports.run = async (browser, url, width, height, fullPage) => {
   }
 
   const keyBase = `${url}-(${width},${height})`
-  const digest = crypto
+  const contentDigest = createContentDigest
```

Comment:
We can remove the chained `.createHash(`md5`).update(keyBase).digest(`hex`)` and replace with 
`createContentDigest({ url, width, height }) `

Reply:
Thank you 😄